lightning/src/chain/channelmonitor.rs

```diff
@@ -3458,9 +3465,11 @@ impl<Signer: WriteableEcdsaChannelSigner> ChannelMonitorImpl<Signer> {
 
 		if height > self.best_block.height() {
 			self.best_block = BestBlock::new(block_hash, height);
+			log_trace!(logger, "Connecting new block {} at height {}", block_hash, height);
 			self.block_confirmed(height, block_hash, vec![], vec![], vec![], &broadcaster, &fee_estimator, logger)
 		} else if block_hash != self.best_block.block_hash() {
 			self.best_block = BestBlock::new(block_hash, height);
+			log_trace!(logger, "New block of block hash {} has been found and updated", block_hash);
```

Comment:
It isn't clear to me from the logging that "updated" means the best block was re-orged out. How about something like the following?

"Best block re-orged, replaced with new block {} at height {}"